package.json

```diff
@@ -40,7 +40,7 @@
     "test:storybook": "test-storybook"
   },
   "dependencies": {
-    "@forgerock/javascript-sdk": "beta",
+    "@forgerock/javascript-sdk": "^4.4.1",
```

Comment:
Should this stay beta, so we can push a updated beta with the changes you need from SDK?

Reply:
we can if we want to publish betas, wasn't sure what approach we were going to take, i just publisehd it locally with that version number